sdk/eventhubs/azure-messaging-eventhubs/src/samples/README.md

```diff
@@ -0,0 +1,42 @@
+# Azure Event Hubs client library samples for Java
```

Comment:
This README.md should be referenced by  README.md in `azure-messaging-eventhubs`  https://github.com/Azure/azure-sdk-for-java/blob/c8360a4bfd902e454a90112a4693401b04b38393/sdk/eventhubs/azure-messaging-eventhubs/README.md
just like we have done in Storage Example : https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-queue/README.md#next-steps-samples 

This was one of the documentation request by Kate from documentation team.

Similar reference should added in https://github.com/Azure/azure-sdk-for-java/blob/c8360a4bfd902e454a90112a4693401b04b38393/sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/README.md

Reply:
One more change  
https://github.com/Azure/azure-sdk-for-java/blob/7689f1fd0b24490f93faf647f38f8ebb1f448beb/sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/README.md

Could we change `sample_examples` link to following, So Samples link point to README instead of .java files. That was another comment from Kate.
[sample_examples]: ./src/samples

Reply:
Done.